src/modrover/learner.py

```diff
@@ -71,6 +64,7 @@ def __init__(
 
     @property
     def opt_coefs(self) -> Optional[np.ndarray]:
+        # TODO: If we have multiple parameters, what's the order of coefficients?
```

Comment:
Important note that I might need clarification on is the order of coefficients in the two parameter model case. Since the parameters are mapped to variables in a dictionary the order might change depending on the parameter type